scripts/qa_crowbarsetup.sh

```diff
@@ -2606,6 +2606,14 @@ function custom_configuration
             fi
             proposal_set_value neutron default "['attributes']['neutron']['use_lbaas']" "true"
 
+            if [[ $want_l3_ha = 1 ]]; then
+                if iscloudver 9M12plus ; then
```

Comment:
NIT: this could be future-proofed already by doing something like: 

``` if grep -q use_l3_ha /opt/dell/chef/data_bags/crowbar/template-neutron.json``` 

then we don't need to update the version detection in lockstep with the backport (I do want to backport the option of deploying with l3_ha also to older releases so that we can more easily test l3-ha on cloud8 and older)

Reply:
Done